packages/block-library/src/navigation/style.scss

```diff
@@ -61,9 +61,8 @@
 		& > li > ul {
 			margin: 0;
 			position: absolute;
-			background: #fff;
```

Comment:
I think it is _correct_ to remove this background, because it is not "light or dark theme aware". 

But it does mean that a theme like TwentyTwenty now looks like this:

<img width="551" alt="Screenshot 2019-11-20 at 08 44 05" src="https://user-images.githubusercontent.com/1204802/69218913-0669dd80-0b72-11ea-845c-85a4bb50b706.png">

Compare that with this before:

<img width="499" alt="Screenshot 2019-11-20 at 08 45 08" src="https://user-images.githubusercontent.com/1204802/69218967-226d7f00-0b72-11ea-8dc1-6a354c75ecb2.png">

This is a tricky one, because the white background assumes a link color that has contrast with that which might not be the case. On the other hand just floating there is also a bit off. Any ideas?

Reply:
Note that I tested TwentyTwenty because it does _not_ opt in to "opinionated styles", and your PR here correctly relegates most of those styles to be actually opinionated.